README.md

```diff
@@ -550,6 +550,24 @@ Expressions should finish quickly or the user experience may drag, especially on
 - [quick-execution](https://angular.io/guide/template-syntax#quick-execution) - official documentation for template expressions
 - [Increasing Performance - more than a pipe dream](https://youtu.be/I6ZvpdRM1eQ) - ng-conf video on youtube. Using pipe instead of function in interpolation expression
 
+### Coalescing event change detections
+
+Angular uses ZoneJS to intercept events that occurred in the application and run a change detection cycle automatically, also Angular runs change detection every time when the microtask queue is empty.
```

Comment:
```suggestion
Angular uses zone.js to intercept events that occurred in the application and runs a change detection automatically. By default this happens when the [microtask queue](https://www.youtube.com/watch?v=cCOL7MC4Pl0) of the browser is empty, which in some cases may call redundant cycles.
```

Reply:
Thanks for picking this up, I'd like to use Jake Archibald's video. I also think it's the best one so far for explaining js event loop.